cpp/log/cert.cc

```diff
@@ -726,6 +730,16 @@ bool IsValidRedactedHost(const string& hostname) {
 }
 
 
+void freeOpenSslStackOfGeneralName(STACK_OF(GENERAL_NAME)* general_name_stack) {
```

Comment:
Fixed the casing of the validateRedactionSubjectAltNames function as well, and moved that into the anonymous namespace.